rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -301,6 +301,8 @@ cloud_resource_warning_vcpus: 80.0
 cloud_resource_critical_vcpus: 90.0
 cloud_resource_warning_disk_space: 80.0
 cloud_resource_critical_disk_space: 90.0
+cloud_resource_cpu_allocation_ratio: 2.0
```

Comment:
No, the way this PR is now structured is that only the nova vars need to be set for an rpco deployment. This in ```user_extras_variables.yml```:

```
cloud_resource_cpu_allocation_ratio: "{{ nova_cpu_allocation_ratio }}"
cloud_resource_mem_allocation_ratio: "{{ nova_ram_allocation_ratio }}"
```

and this in ```user_variables.yml```

```
nova_cpu_allocation_ratio: 2.0
nova_ram_allocation_ratio: 1.0
```

allow that to happen